src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,29 @@
-export const PostInfo = () => <>Put the post here</>;
+import React from 'react';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={post.user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      <hr />
+
+      {post.comments.length !== 0 ? (
```

Comment:
```suggestion
      {!!post.comments.length ? (
```

Reply:
Redundant double negation.eslint[no-extra-boolean-cast](https://eslint.org/docs/rules/no-extra-boolean-cast)

Reply:
ok, let it be